src/test/java/com/intive/patronage22/szczecin/retroboard/service/BoardServiceTest.java

```diff
@@ -342,7 +338,8 @@ void patchBoardShouldReturnUserIsNotAnOwner() {
         final var email = "username@test.pl";
         final var user = new User(uid, email, "displayName", Set.of());
         final var id = 10;
-        final var board = buildBoard(user);
+        final var state = EnumStateDto.CREATED;
```

Comment:
you don't need to create state as separate variable if it's used only in line below
```suggestion
        final var board = buildBoard(user, EnumStateDto.CREATED);
```